testing/ostest/tls.c

```diff
@@ -35,8 +35,6 @@
 
 #if CONFIG_TLS_NELEM > 0
 
-#include <arch/tls.h>
```

Comment:
Don't we need to include `<nuttx/arch.h>` now?

Reply:
Not needed because `include/nuttx/tls.h` already includes it.